pennylane/plugins/default_gaussian.py

```diff
@@ -852,30 +853,30 @@ def reset(self):
         # init the state vector to |00..0>
         self._state = vacuum_state(self.num_wires, self.hbar)
 
-    def reduced_state(self, registers):
+    def reduced_state(self, wire_indices):
         r""" Returns the vector of means and the covariance matrix of the specified wires.
 
         Args:
-            registers (Iterable[int] or int): indices of the requested wires
+            wire_indices (Iterable[int] or int): indices of the requested wires
 
         Returns:
             tuple (means, cov): means is an array containing the vector of means,
             and cov is a square array containing the covariance matrix
         """
-        if registers == list(range(self.num_wires)):
+        if wire_indices == list(range(self.num_wires)):
             # reduced state is full state
             return self._state
 
         # reduce rho down to specified subsystems
-        if isinstance(registers, int):
-            registers = [registers]
+        if isinstance(wire_indices, int):
+            wire_indices = [wire_indices]
 
-        if np.any(np.array(registers) > self.num_wires):
+        if np.any(np.array(wire_indices) > self.num_wires):
             raise ValueError(
                 "The specified wires cannot " "be larger than the number of subsystems."
             )
 
-        ind = np.concatenate([np.array(registers), np.array(registers) + self.num_wires])
+        ind = np.concatenate([np.array(wire_indices), np.array(wire_indices) + self.num_wires])
         rows = ind.reshape(-1, 1)
         cols = ind.reshape(1, -1)
 
```

Comment:
I think one of the hoped-for benefits of a wire management class is that both the user, and the plugin developer, would only have to worry about how the device works and not any bookkeeping. However, that doesn't really work in this particular plugin, since (correct me if I misunderstand) some functions need to know both about the "user" wires and the "device" register :thinking:

Reply:
Yes, exactly. I will reply to your general comment about that.

Reply:
Not really (if I understand your comment correctly), the plugin developer needs to know that the user creates a device with a register of custom labels, and that one can get the index of a subsystem using the `indices_on_register` function. Nothing needs to be known about the actual labels. 

A developer could also force the device to have a fixed register, in which case s/he has control over the labels as well, for example where a hardware comes with specific wire names.

Reply:
Even though most devices just work on the `wire_indices`, this system makes sure that this object is very controlled, it is always a list of integers addressing subsystems that exist on the device.

Reply:
In this case it is because the ``poly_quad_expectations`` is not a method of the CV operator class. So it cannot just get the attribute. It's more a helper function...